Raven.NET.Core/Observers/RavenTypeWatcher.cs

```diff
@@ -17,6 +18,8 @@ public class RavenTypeWatcher : IRavenTypeWatcher
     {
         private readonly IRavenProvider _ravenProvider;
         private readonly IRavenSettingsProvider _ravenSettingsProvider;
+        
+        private readonly IRavenStorage _ravenStorage = RavenStorage.Instance;
```

Comment:
I think there you can inject it via dependency injection, as well as in other watcher class, in tests i think you will always have to refer to instance